specs/2024-10-30.yaml

```diff
@@ -2457,27 +2456,12 @@ components:
           description: |-
             The unique identifier for the payment that the transaction is associated with.
           example: "pay_1LaXpKGUcADgqoEMl0Cx0Ygg"
-        payment:
-          $ref: "#/components/schemas/Payment"
-        payment_details:
-          $ref: "#/components/schemas/PaymentDetails"
         attachment_file_id:
           type: string
           description: |-
             The unique identifier for the file that is attached to the transaction.
             This is useful if you want to attach a letter or another document to the transaction.
           example: "file_1LaXpKGUcADgqoEMl0Cx0Ygg"
-        status:
-          type: string
-          description: The status of the transaction
-          example: "pending"
-          readOnly: true
-          enum:
-            - pending
-            - submitted
-            - completed
-            - canceled
-            - rejected
```

Comment:
since we dont allow for updating a transaction, I didnt think it made sense to provide a status (which inherently implies it should be updatable)